desktop/ui/pom.xml

```diff
@@ -72,28 +67,6 @@
 			<artifactId>DataCleaner-env-berkeleydb</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-
-		<!-- DataCleaner components -->
-		<dependency>
-			<groupId>org.eobjects.datacleaner</groupId>
-			<artifactId>DataCleaner-standard-components</artifactId>
```

Comment:
OK but I think it's best to explicitly name it here. The transitive dependency of xml-config is more a coincidence than design, whereas "ui" depending on "api" is certainly a strong contract.